src/java.base/share/classes/java/io/DataInputStream.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1994, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1994, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Only copyright year is updated in this file?

Reply:
The previously proposed changes were rolled back. I will revert the year back to 2022.